app/controllers/miq_ae_class_controller.rb

```diff
@@ -2729,6 +2733,11 @@ def flash_validation_errors(am_obj)
     end
   end
 
+  def add_active_node_to_open_nodes
+    return unless @sb.dig('trees') && @sb.dig('trees').dig('ae_tree') && @sb.dig('trees').dig('ae_tree').dig('open_nodes')
```

Comment:
@PanSpagetka by using `#dig`, you don't need to investigate each node separately

```ruby
return unless @sb.dig('trees', 'ae_tree', 'open_nodes').present?
```